lib/ui/platform_dispatcher.dart

```diff
@@ -2605,3 +2621,37 @@ class SemanticsActionEvent {
     );
   }
 }
+
+/// Callback to subscribe to [ViewFocusChange] notifications.
+typedef ViewFocusChangeCallback = void Function(ViewFocusChange);
+
+/// An event to communicate view focus changes between the framework and engine.
+///
+/// If the event comes from the framework it means there was a flutter focus change.
```

Comment:
nit: s/flutter/Flutter/
nit: the word "flutter" includes both the framework and the engine, so it's not clear what "there was a Flutter focus change means". Suggestion:

```
/// When provided from the framework to the engine, indicates a request
/// that the engine change the focus state of a view.
```

However, I'm warming up to the idea of not using this type for `PlatformDispatcher.requestViewFocus`.

Reply:
Ah, yes! See also one of my comments above about using the term "flutter" here.

Reply:
Great feedback. Changed!